integration/app/test-demo/README.md

```diff
@@ -0,0 +1,81 @@
+# README - React Testing Demo
+
+The React Testing Demo is used to test the Amazon Chime SDK React Components Library.
+
+Different tests tend to use different setups. Creating a separate test application for each setup would not make sense and would add additional maintenance work. Following the example of [WebRTC samples](https://webrtc.github.io/samples/), we come up with the concept of a Mini Test App - a minimalist, single-page application containing only the necessary components for a single test. All Mini Test Apps are sub-pages of the React Testing Demo.
+
+## Guideline
+
+When adding tests for a functionality, build a new Mini Test App.
+
+* For example, we already have a "Roster Test App" and you want to add tests for the video feature. You should create a "Video Test App".
```

Comment:
`For example, we already have a "Roster Test App" and you want to add tests for video features, you should create a "Video Test App".`

Reply:
Updated.